### PR TITLE
New features proposed in issues implemented in the ontology

### PR DESCRIPTION
**Pull request by anaigmo**
This PR includes new features in the ontology and related documentation, solving the following issues:
* [#120](https://github.com/kg-construct/rml-core/issues/120): Should DatatypeMap be defined as a TermMap?
* [#72](https://github.com/kg-construct/rml-core/issues/72) add `rml:UnsafeIRI` and `rml:UnsafeURI` to ontology, shapes and spec
* [#50](https://github.com/kg-construct/rml-core/issues/50) add `rml:URI` to ontology, shapes and spec
* [#30](https://github.com/kg-construct/rml-core/issues/30) add `rml:baseIRI` property to ontology, shapes and spec
* [#51](https://github.com/kg-construct/rml-core/issues/51) Move `rml:Strategy`and `rml:strategy` to RML-CC 
*  [#51](https://github.com/kg-construct/rml-io/issues/61) Inconsistency due to rdfs:domain of rml:logicalTarget and its use on rml:LanguageMap

### CODE REVIEW

**Review by pmaria — CHANGES_REQUESTED** [review]
Thanks @anaigmo. Some minor comments

File: ontology/rml-core.owl
Comment (pmaria) [inline ontology/rml-core.owl:96]:
```suggestion
       rdfs:comment "Indicates the base IRI of the RDF graph to be created with this triples map."@en ;
```

**Comment by anaigmo** [comment]
Updated files according to @pmaria 's review in last commit

**Comment by dachafra** [comment]
ok, let's wait until his final approval to merge it (@pmaria), thanks @anaigmo

File: ontology/rml-core.owl
Comment (pmaria) [inline ontology/rml-core.owl:96]: Within is a bit odd, with is better since it indicates that this triples map is used to generate a graph **_with_**.

```suggestion
       rdfs:comment "Indicates the base IRI of the RDF graph to be created with a triples map."@en ;
```

**Review by pmaria — CHANGES_REQUESTED** [review]
Just one minor comment on the definition of baseIRI. Once that is fixed we can merge it

**Comment by anaigmo** [comment]
Oki, now that definition is fixed :)

**Review by pmaria — APPROVED** [review]

**Comment by chrdebru** [comment]
I looked at @anaigmo and @pmaria's work in the spec and shapes. I just want to confirm, unlike @dachafra's proposal of "pushing it further", `rml:baseIRI`'s domain is triples maps, right?

Also, do we keep base IRIs as input in the configuration (overwritten by `rml:baseIRI`) when present, or do we assume that baseIRIs must be provided. The former is more flexible. The latter will require us to change all the test cases and examples in our specifications.

**Comment by pmaria** [comment]
> I looked at @anaigmo and @pmaria's work in the spec and shapes. I just want to confirm, unlike @dachafra's proposal of "pushing it further", `rml:baseIRI`'s domain is triples maps, right?

Yest that is how I understood it, and how it is defined in the ontology now.

> Also, do we keep base IRIs as input in the configuration (overwritten by `rml:baseIRI`) when present, or do we assume that baseIRIs must be provided. The former is more flexible. The latter will require us to change all the test cases and examples in our specifications. Happy to work on `rml:baseIRI` test cases in the case of the former (testing failures, testing overwrites, ...).

Yes, the former would be my preference as well. I think we should also explicitly define a default in the spec for if no baseIRI is provided via configuration.

**Pull request merged**
